### PR TITLE
Add reset script

### PR DESCRIPTION
**Pull request by dcmcand**
**Description of change**
Adds a small convenience script i use when changing branches.

This script: 
1. stops all running containers
2. deletes current docker volumes
3. reinstalls deps
4. runs migrations
5. runs seeders
It makes sure you have a clean environment when switching branches.

**Caution**
This script removes all docker volumes not attached to a running container. If you have other docker volumes that you are not ok with deleting, do not run this script.

**How to test**
run `yarn docker:reset`

### CODE REVIEW

**Review by jasalisbury — COMMENTED** [review]
Looks good, but some pre-signed url stuff got in as well. Will either want to update this PR or wait for the pre-signed stuff to make it to main.

**Comment by dcmcand** [comment]
Removed the presigned url stuff.

**Review by jasalisbury — APPROVED** [review]

File: bin/reset-all [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+yarn docker:stop
+
+docker volume prune
```
Comment (rahearn) [inline bin/reset-all:5]: These two could be combined as `yarn docker:stop --volumes` which would also have the benefit on not removing unrelated volumes.

**Review by rahearn — CHANGES_REQUESTED** [review]
Let's make the change to `down` so we don't accidentally hose a future developer's volumes.

**Review by rahearn — APPROVED** [review]

**Pull request merged**